hub/templates/hub/area/_place_data.html

```diff
@@ -1,131 +1,152 @@
+                        <th>{{ range.label }}</th>
+                        <td>{{ range.value|floatformat }}%</td>
+                        <td class="text-muted">{{ range.average|floatformat }}%</td>
+                    </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+                {% elif category.data.data_type.name == 'constituency_popular_petitions' %}
+                    <table class="table mb-0">
+                        <thead>
+                            <tr>
+                                <th scope="col">Petition</th>
+                                <th scope="col">Signatures</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                        {% for petition in category.data.value %}
+                            <tr>
+                                <td><a href="{{ petition.url }}">{{ petition.action }}</a></td>
+                                <td class="text-end">{{ petition.signatures }}</td>
+                            </tr>
+                        {% endfor %}
+                        </body>
+                    </table>
+                {% elif category.name == 'Air pollution' %}
+                    <table class="table mb-0">
+                        <thead>
+                            <tr>
+                                <th scope="col">Pollutant</th>
+                                <th scope="col">This area</th>
+                                <th scope="col" class="text-muted">UK average</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {% for range in category.data %}
+                            <tr>
+                                <th>
+                                    {{ range.label|html_format_dataset_name|safe }}
+                                {% if range.data_type.description %}
+                                    <small class="d-block mt-1 fs-8 fw-normal">{{ range.data_type.description|escape }}</small>
+                                {% endif %}
+                                </th>
+                                <td>{{ range.value|floatformat }}%</td>
+                                <td class="text-muted">{{ range.average|floatformat }}%</td>
+                            </tr>
+                            {% endfor %}
+                        </tbody>
+                    </table>
+                {% elif category.name == 'Flood risk from rivers or sea' %}
+                        <table class="table mb-0 js-chart" data-chart-type="bar" data-chart-direction="y"">
+                                <thead>
+                                    <tr>
+                                        <th scope="col">Risk</th>
+                                        <th scope="col" data-color="#068670">This area</th>
+                                        <th scope="col" data-color="#ced4da">UK average</th>
```

Comment:
This should be England not UK?

Reply:
This is an issue for a number of datasets that are England or England+Wales only. I figure it’s something we chalk up to fix later, rather than holding up this dataset import.